react/features/toolbox/components/web/ToolbarButton.js

```diff
@@ -4,6 +4,7 @@ import React from 'react';
 
```

Comment:
seems like this component is only being used by ReactionsMenuButton and OverflowMenuButton. Can't those just inherit the AbstractButton instead?

Reply:
Yep removed ToolbarButton.js completely and switched to using AbstractButton for the 2.